build.gradle.kts

```diff
@@ -15,6 +15,7 @@ buildscript {
 plugins {
     id("appyx-collect-sarif")
     id("com.autonomousapps.dependency-analysis") version libs.versions.dependencyAnalysis.get()
+    id("release-dependencies-diff") apply false
```

Comment:
Optional: have 2 plugins – a root plugin that will register `releaseDependenciesDiffFiles` and apply to all subprojects and a child plugin that will register `releaseDependenciesCreateFiles`.

Reply:
done